src/foam/core/FObjectArray.js

```diff
@@ -22,6 +22,48 @@ foam.CLASS({
 
   documentation: "A Property which contains an array of 'of' FObjects.",
 
+  methods: [
+    function installInProto(proto) {
+      this.SUPER(proto);
+      var self = this;
+      Object.defineProperty(proto, self.name + '$' + 'errors', {
+        get: function getArrayErrorSlot() {
+          // errors_$ will be updated with an array member's error_$ slot
+          var errors_$ = foam.core.SimpleSlot.create({ value: [] });
+          // sub will be replaced when the array changes
+          var sub = foam.core.FObject.create();
+
+          // Safely combine error lists and update error_$
+          var setErrors = errorLists => {
+            errors_$.set(errorLists
```

Comment:
Could just be: errorLists.flat().map(v=>v) ?

Reply:
I want this to be consistent with checking `errors_` on an FObject. The `map` is used to turn nulls into arrays so that `concat` doesn't add nulls as array elements (there are places where errors are checked by seeing if the array length is greater than zero)

For example this outputs `[null, null]` which would indicate there are errors to display (list length > 0)
```
[null, null].flat()
```

but this outputs `[]` which is what I want it to do
```
[null, null].map(v => v || []).flat()
```

Reply:
[null,null].flat().filter(v=>v);
also gives [] but with 1 fewer character ;-)

Reply:
Oh, my brain flipped the filter to a map for some reason

Reply:
fixed!